src/aks-preview/azext_aks_preview/tests/latest/test_validators.py

```diff
@@ -179,7 +179,7 @@ def test_throws_on_negative(self):
 
 class TestSpotMaxPrice(unittest.TestCase):
     def test_valid_cases(self):
-        valid = [5, 5.12345, -1.0]
+        valid = [5, 5.12345, -1.0, 0.068, 0.071, 5.00000000]
```

Comment:
Is 5.00000000 valid? According to your new code, length('00000000') > 5.

Reply:
when it gets to the function, it becomes 5.0

Reply:
in python, float number with all 0 after dot will be reduced to .0

Reply:
I see. It's a number, not a string.